tests/configuration_spec.lua

```diff
@@ -54,6 +54,43 @@ describe("configuration", function()
         })
     end)
 
+    it("can define and use multi-byte mappings", function()
+        require("nvim-surround").setup({
+            surrounds = {
+                -- multi-byte linguistic quote
+                ["’"] = {
+                    add = { "’", "’" },
+                    delete = "^()().-()()$",
```

Comment:
I'm honestly a bit confused as for how this unit test is working, since it seems like you're not actually putting anything into the capture groups *to* be deleted. It should be `^(.)().-(.)()$` instead.

Reply:
Oh nevermind the unit test simply isn't passing :grinning:

Reply:
I agree, I added `’` where relevant, should make more sense now